doc/en/builtin.rst

```diff
@@ -162,7 +162,7 @@ For information about fixtures, see :ref:`fixtures`. To see a complete list of a
 
     no tests ran in 0.12 seconds
 
-You can also interactively ask for help, e.g. by typing on the Python interactive prompt something like::
+You can also interactively ask for help, e.g. by typing on the Python interactive prompt something like
```

Comment:
Oh sorry I should have been more clear, the `::` should be replaced by `:`, as the `:` makes gramatical sense.